components/Hits.js

```diff
@@ -7,13 +7,13 @@ class Hits extends React.Component {
   renderWithResults() {
     let renderedHits = map(this.props.results.hits, hit => {
       return (
-        <div className={this.props.cssClasses.item} key={hit.objectID}>
```

Comment:
No my bad, should be ok: http://facebook.github.io/react/docs/multiple-components.html#dynamic-children the example is using the same notation than us